config/prod.webpack.config.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable max-len */
+const webpack = require('webpack');
+const { resolve } = require('path');
+const config = require('@redhat-cloud-services/frontend-components-config');
+const { config: webpackConfig, plugins } = config({
+  rootFolder: resolve(__dirname, '../'),
+  debug: true,
+});
+
+plugins.push(
+  require('@redhat-cloud-services/frontend-components-config/federated-modules')(
+    {
+      root: resolve(__dirname, '../'),
```

Comment:
You are not exposing the same files as in the dev config.

Reply:
Good catch!